polkadot/xcm/xcm-simulator/fuzzer/src/relay_chain.rs

```diff
@@ -259,3 +265,53 @@ construct_runtime!(
 		MessageQueue: pallet_message_queue,
 	}
 );
+
+#[sp_version::runtime_version]
+pub const VERSION: RuntimeVersion = RuntimeVersion {
+	spec_name: create_runtime_str!("node-template"),
```

Comment:
this doesn't sound like the right name to me. 

```suggestion
	spec_name: create_runtime_str!("xcm-simulator-node"),
```

Reply:
This was removed in 3281a1f.